test/sql/updates/test-0.1.1.sql

```diff
@@ -0,0 +1 @@
+\d+ _timescaledb_catalog.hypertable;
```

Comment:
Shouldnt this also have \d+ _timescaledb_catalog.hypertable_index as the structure of that also changed?